extensions/cnsr_ext.lua

```diff
@@ -175,6 +176,13 @@ function load_tags_from_file()
 		vlc.osd.message("Failed to load cnsr file", nil, "bottom-right")
 		return nil
 	end
+
+	if not valid_cnsr_file(cnsr_file) then
+		vlc.osd.message("cnsr file bad format", nil, "bottom-right")
+		Log("cnsr file isn't valid")
+		return nil
+	end
+
```

Comment:
io.close(cnsr_file)
cnsr_file = io.open(cnsr_uri,"r")

we need to reset the line iterator